eng/targets/CSharp.Common.props

```diff
@@ -1,7 +1,7 @@
 <Project>
 
   <PropertyGroup>
-    <LangVersion>8.0</LangVersion>
+    <LangVersion>preview</LangVersion>
```

Comment:
Only thing you can do as a temporary workaround would be to change this to 8.0 until we get a new SDK.

Reply:
That resulted in errors later in the build when something hit an unsupported expression. I'll try again though.